tests_e2e/tests/lib/vm_extension_identifier.py

```diff
@@ -34,7 +34,7 @@ def __init__(self, publisher: str, ext_type: str, version: str):
 
     unsupported_distros: Dict[str, List[str]] = {
         "Microsoft.OSTCExtensions.VMAccessForLinux": ["flatcar"],
-        "Microsoft.Azure.Monitor.AzureMonitorLinuxAgent": ["flatcar"]
+        "Microsoft.Azure.Monitor.AzureMonitorLinuxAgent": ["flatcar", "mariner_1"]
```

Comment:
get_distro.py produces <distro>_<version> and here we are only using the name ("flatcar")

Reply:
We check if any of these are a substring in the result of get_distro.py. This check is on Line 45

Reply:
If we only put distro name in this list, then we know that extension doesn't support any version of the distro. If we put <name>_<version> then we can mark that the extension only excludes a certain version. I did this since mariner_1 is unsupported by AMA, but mariner_2 is supported

Reply:
where do we strip the version so that "flatcar" matches the value returned by get_distro.py?